docs/DeveloperGuide.md

```diff
@@ -654,8 +654,51 @@ Names in should only contain alphanumeric characters
 Example: listPerson Alex David
 ```
 
-### Improved Space Sensitivity in Names
+### Stronger Email Input Validation
+- **Background**: Currently, the `Email` of our `Person` is currently validated
```

Comment:
Currently is repeated (maybe delete one of the 'currently's)

Reply:
Changed